RequestService/RequestService.Handlers/GetRequestDetailsHandler.cs

```diff
@@ -0,0 +1,41 @@
+﻿using MediatR;
+using System.Threading;
+using System.Threading.Tasks;
+using RequestService.Core.Interfaces.Repositories;
+using HelpMyStreet.Contracts.RequestService.Request;
+using Microsoft.Extensions.Options;
+using RequestService.Core.Config;
+using HelpMyStreet.Contracts.RequestService.Response;
+using RequestService.Core.Services;
+
+namespace RequestService.Handlers
+{
+    public class GetRequestDetailsHandler : IRequestHandler<GetRequestDetailsRequest, GetRequestDetailsResponse>
+    {
+        private readonly IRepository _repository;
+        private readonly IJobService _jobService;
+        private const int ADMIN_USERID = -1;
+        public GetRequestDetailsHandler(IRepository repository, IJobService jobService)
+        {
+            _repository = repository;
+            _jobService = jobService;
+        }
+
+        public async Task<GetRequestDetailsResponse> Handle(GetRequestDetailsRequest request, CancellationToken cancellationToken)
+        {
+            bool hasPermission = true;
+            GetRequestDetailsResponse response = null;
+
+            //if (request.AuthorisedByUserID != ADMIN_USERID)
```

Comment:
For now, let's allow TaskAdmins on the referring group.  I don't think we need to allow ADMIN_USERID.  I also don't think we need to allow users allocated to the jobs; they should be generating requests to a different endpoint which accepts JobID rather than RequestID.

Reply:
(VRRM-33)